Helpers/helpers.ts

```diff
@@ -0,0 +1,32 @@
+export const checkboxHitSlop = { bottom: 20, left: 20, right: 20, top: 20 };
+
+export const fullDayTimes = [
```

Comment:
I would probably re-write this as a loop to generate this. Would be pretty easy and look a lot better